src/main/java/com/puppycrawl/tools/checkstyle/checks/TrailingCommentCheck.java

```diff
@@ -102,23 +102,34 @@
  * &lt;module name=&quot;TrailingComment&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * // OK
+ * if (&#47;&#42; OK &#42;&#47; x &#62; 5) {}
+ * int a = 5; // violation, string before comment should match pattern "^[\s});]*$"
+ * doSomething(
+ *   param1
+ * ); // OK
```

Comment:
`// OK , se we by default such trailing of method/code-block ending is allowed`

Reply:
I'm sorry, `OK,  se we by default..`? Did you mean `OK, by default..`?

Reply:
Done!